lenskit-api/src/main/java/org/grouplens/lenskit/RecommenderEngine.java

```diff
@@ -39,5 +39,5 @@
      *
      * @return A recommender ready for use.
      */
-    public Recommender createRecommender();
```

Comment:
Cleanup and churn - interface methods are implicitly public, so `public` keyword is unnecessary.